Working

buland-usgs opened the pull request:
Rebase with John's JavaDocs changes plus JavaDocs comment updates.  Also, continuing active development on the travel-time table generation.

jpatton-USGS commented:
Hey Ray, the Travis CI build fails for this project. Its Looking for various components of org.apache.commons.math3. Is this a new library jar we need to add to the project and build.xml?

Also, why are you adding /doc/ to gitignore, there are source controlled files in there (index.md for the documentation site, in future the user manual, etc.)

buland-usgs commented:
John,
   I just started on a new piece of the project and I am indeed using
Apache commons.  I'll add it to the library directory so that it will show
up on GitHub.
   The docs thing is confusing to me.  Since you're generating JavaDocs on
the server, I thought I should quit generating JavaDocs at my end and
pushing them to GitHub.  That created a problem on my end since there are
so many html files that I can't see what I changed.  That's when I added
doc/ to .gitignore.  I compensated by turning on the real time JavaDocs
error checking in Eclipse (for some reason it's off by default) so that the
JavaDocs generated on the server would be right.  Maybe I should delete
doc/ from .gitignore and just delete the JavaDocs files from doc/ on my
local machine.  That would at least eliminate conflicts if I have to rebase
again.  I'm really not sure how this all works.  What do you suggest?

Ray

On Wed, Apr 25, 2018 at 8:27 AM, John Patton <notifications@github.com>
wrote:

> Hey Ray, the Travis CI build fails for this project. Its Looking for
> various components of org.apache.commons.math3. Is this a new library jar
> we need to add to the project and build.xml?
>
> Also, why are you adding /doc/ to gitignore, there are source controlled
> files in there (index.md for the documentation site, in future the user
> manual, etc.)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/usgs/neic-traveltime/pull/22#issuecomment-384306693>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AVksQ8w8cUIhOOB7VhADUkno3wc5GaSxks5tsIfSgaJpZM4TiiZn>
> .
>



-- 
                                                      Ray Buland

buland-usgs commented:
John,
   OK.  I made it so.  When I get the blurbs written for the two projects,
should I just drop them into doc/?

Ray

On Wed, Apr 25, 2018 at 11:23 AM, Ray Buland <rbjoland@gmail.com> wrote:

> John,
>    OK.  I'll make it so.
>
> Ray
>
> On Wed, Apr 25, 2018 at 11:19 AM, John Patton <notifications@github.com>
> wrote:
>
>> Hey Ray,
>> I'd favor deleting the javadocs files from doc/ locally or generating
>> them locally into a different temporary location than doc (javadocs/
>> maybe?). My intention is to use doc/ for things like the user guide, so it
>> would be nice to not .gitignore it.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/usgs/neic-traveltime/pull/22#issuecomment-384365763>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AVksQ4zqhSdgs0_7D8rrlr1EESfbZo-kks5tsLAYgaJpZM4TiiZn>
>> .
>>
>
>
>
> --
>                                                       Ray Buland
>



-- 
                                                      Ray Buland

jpatton-USGS commented:
Ray, I fixed the build.xml file for the new library, and added it's license file.

Depending on their length, I was going to drop them into the main project readme.md files or in the doc directory if they were longer than a couple paragraphs. Either way I'd want to reformat them into .md (git markdown format) files, so just emailing them to me would probably be easiest.

buland-usgs commented:
John,
   OK.  No problem.

Ray

On Wed, Apr 25, 2018 at 11:40 AM, John Patton <notifications@github.com>
wrote:

> Ray, I fixed the build.xml file for the new library, and added it's
> license file.
>
> Depending on their length, I was going to drop them into the main project
> readme.md files or in the doc directory if they were longer than a couple
> paragraphs. Either way I'd want to reformat them into .md (git markdown
> format) files, so just emailing them to me would probably be easiest.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/usgs/neic-traveltime/pull/22#issuecomment-384372431>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AVksQ7YnyBXzX8O2bsBHDC2Qa6KXGZuDks5tsLUMgaJpZM4TiiZn>
> .
>



-- 
                                                      Ray Buland
